tools/listErrorsWithoutCWE.py

```diff
@@ -12,9 +12,8 @@ def main():
     tree = ET.parse(vars(parsed)["F"])
     root = tree.getroot()
     for child in root.iter("error"):
-
         if "cwe" not in child.attrib:
-            print child.attrib["id"], ",", child.attrib["severity"], ",", child.attrib["verbose"]
+            print(child.attrib["id"], child.attrib["severity"], child.attrib["verbose"], sep=", ")
```

Comment:
This doesn't print the same, but I think new output is better:
Before: `a , b , c`
After: `a, b, c`